test/modules/TestResultCoordinator/Reports/TestReportMetadataBase.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports
+{
+    public abstract class TestReportMetadataBase
+    {
+        public TestReportMetadataBase(string testDescription)
+        {
+            this.TestDescription = testDescription;
```

Comment:
Please implement TestReportType and TestOperationResultType as well.

Reply:
I tried doing this initially, but thought it was a bad idea due to misunderstanding with properties. Figured it out though and made the change.